synbiohub_adapter/upload_sbol/upload_sbol.py

```diff
@@ -58,7 +58,7 @@ def main(args=None):
 class SynBioHub():
     def __init__(self, url, email, password, sparql):
         self.url = url
-        self.part_shop = PartShop(url + '/')
+        self.part_shop = PartShop(url)
```

Comment:
Hi @tcmitchell, it was originally my plan to fix the problem at its source, in pySBOL, per my comment on Feb 20th.  But it is easy enough to put a safeguard in synbiohub_adapter, too, which I did in 
bf6f821

Reply:
It's such a trivial pull request and yet...

I think [strip](https://docs.python.org/3/library/stdtypes.html#str.strip) isn't what you want there since it removes the given characters from both the beginning and the end of the string. [rstrip](https://docs.python.org/3/library/stdtypes.html#str.rstrip) does what you really want. And it turns out there's no need to do the check for a trailing slash, `rstrip` handles that automatically. So you could just do:

```
# PartShop fails if there is a trailing slash in the URL
self.part_shop = PartShop(url.rstrip('/'))
```